firefox-ios/Client/Application/SceneDelegate.swift

```diff
@@ -56,14 +56,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Handle clean-up here for closing windows on iPad
         guard let sceneCoordinator = (scene.delegate as? SceneDelegate)?.sceneCoordinator else { return }
 
-        // Give all coordinators a chance to respond to scene disconnect (window close).
-        sceneCoordinator.recurseChildCoordinators {
-            ($0 as? WindowEventCoordinator)?.coordinatorWindowWillClose()
-        }
-
         // Notify WindowManager that window is closing
-        let windowManager: WindowManager = AppContainer.shared.resolve()
-        windowManager.windowDidClose(uuid: sceneCoordinator.windowUUID)
+        (AppContainer.shared.resolve() as WindowManager).windowWillClose(uuid: sceneCoordinator.windowUUID)
```

Comment:
Are there any advantages to put this on a one-liner?

Reply:
Not that I know of, apart from using 1 less line. Storing in a local variable didn't seem necessary but I believe the compiled code will be identical; I'm happy to change it back if you have a preference.

Reply:
🤷‍♀️ both works for me, was just curious. Thank you!